src/Pages/Auth/Register.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Can you remove this file and the login component. Its not related to this card

Reply:
Ohk